demo-code/requirements.in

```diff
@@ -7,4 +7,9 @@ whitenoise
 azure-keyvault-secrets
 azure-identity
 opencensus-ext-azure
-opencensus-ext-django
\ No newline at end of file
+opencensus-ext-django
+pytest
```

Comment:
This brings in dev requirements for the app itself, making the container image larger, right? Why not leave in requirements-dev?